<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Azure Remote Rendering Toolkit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="Tabs">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Upload">
       <attribute name="title">
        <string>Storage</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="StorageBrowserWidget" name="StorageBrowser" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Convert">
       <attribute name="title">
        <string>Conversion</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="ConversionWidget" name="ConversionTab" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Render">
       <attribute name="title">
        <string>Rendering</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="RenderingWidget" name="RenderingTab" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Log">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="LogWidget" name="LogTab" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StorageBrowserWidget</class>
   <extends>QWidget</extends>
   <header location="global">Storage/UI/StorageBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConversionWidget</class>
   <extends>QWidget</extends>
   <header location="global">Conversion/UI/ConversionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenderingWidget</class>
   <extends>QWidget</extends>
   <header location="global">Rendering/UI/RenderingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header location="global">App/LogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Tabs</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/ArrtApplication.qrc"/>
 </resources>
 <connections/>
</ui>
